seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -765,7 +765,7 @@
       "title": "Nest an Anchor Element within a Paragraph",
       "description": [
         "Again, here's a diagram of an <code>a</code> element for your reference:",
-        "<a href=\"//i.imgur.com/hviuZwe.png\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" title=\"Click to enlarge\" alt=\"a diagram of how anchor tags are composed with the same text as on the following line\" src=\"//i.imgur.com/hviuZwe.png\"></a>",
+        "<a target=\"_blank\" href=\"//i.imgur.com/hviuZwe.png\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" title=\"Click to enlarge\" alt=\"a diagram of how anchor tags are composed with the same text as on the following line\" src=\"//i.imgur.com/hviuZwe.png\"></a>",
```

Comment:
@rarmatei Awesome! :blush:

Just some random thoughts, but I guess `data-lightbox="img-enlarge"` is used to open images in a lightbox. But it doesn't seem to be used currently. It seems like someone had the intention to ensure that clicking images opens a larger version, staying on the same page. Maybe that would be even better UX than opening the link in a new tab?

For your second question, I'm not sure. @freeCodeCamp/moderators, what do you think?